src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareRecommendationIT.kt

```diff
@@ -1,48 +1,35 @@
 package com.atlassian.performance.tools.hardware
 
-import com.amazonaws.services.ec2.model.InstanceType.*
+import com.amazonaws.services.ec2.model.InstanceType.C59xlarge
+import com.amazonaws.services.ec2.model.InstanceType.M44xlarge
 import com.amazonaws.services.s3.transfer.TransferManagerBuilder
 import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.aws
 import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.taskName
 import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.workspace
-import com.atlassian.performance.tools.hardware.guidance.JiraExplorationGuidance
+import com.atlassian.performance.tools.hardware.guidance.SingleHardwareGuidance
 import com.atlassian.performance.tools.infrastructure.api.distribution.PublicJiraSoftwareDistribution
 import com.atlassian.performance.tools.lib.s3cache.S3Cache
 import com.atlassian.performance.tools.lib.workspace.GitRepo2
-import com.atlassian.performance.tools.virtualusers.api.TemporalRate
 import org.eclipse.jgit.api.Git
 import org.junit.Test
 import java.io.File
-import java.time.Duration
 
 class HardwareRecommendationIT {
 
     @Test
     fun shouldRecommendHardware() {
         requireCleanRepo()
-        val jswVersion = System.getProperty("hwr.jsw.version") ?: "7.13.0"
+        val jswVersion = "8.1.0"
```

Comment:
A more general question.

Is there a reason the same tests is constantly re-written rather than simply adding new tests? Don't we lose, or at least make it more difficult to see, the history of what has been run?